### PR TITLE
Proxy apphosts through death

### PR DESCRIPTION
**Pull request by Kalissaac**
Still need to figure out how we want to handle groups who want to switch to apphosting, since our current method of adding them to `vhost-app.conf` will cause their development domain to be promoted to production immediately even if they're still working on developing their app or configuring systemd. Maybe we initially add `group-studentorg-berkeley-edu.apphost.ocf.berkeley.edu` as their domain and then edit it to `group.studentorg.berkeley.edu` when they're ready to promote the site to production?

### CODE REVIEW

File: modules/ocf_www/files/vhost-web.jinja
Comment (oliver-ni) [inline modules/ocf_www/files/vhost-web.jinja:21]: Don't believe this line is necessary

```suggestion
```

**Review by oliver-ni — APPROVED** [review]

**Comment by oliver-ni** [comment]
Probably wait on rjz approval not entirely familiar w Apache

**Review by singingtelegram — APPROVED** [review]
send it

**Comment by Kalissaac** [comment]
We need to hardcode death's IPs since nginx is too old to support hostnames in `set_real_ip_from` :/

**Pull request merged**
